vega_query/visualisations/overlay.py

```diff
@@ -894,6 +893,50 @@ def overlay_volume(
         y.append(price * size if price != 0 else np.nan)
     ax.step(x, y, label="volume", where="post")
 
+def overlay_cumulative_volume(
+    ax: Axes,
+    trades: List[protos.vega.vega.Trade],
+    price_decimals: int,
+    size_decimals: int,
```

Comment:
Small change that some (but not all) of the overlays do. Lets you pass mpl kwargs to modify the line plot.

```suggestion
    size_decimals: int,
    **kwargs,
```